src/net.h

```diff
@@ -359,15 +360,13 @@ class CNode
 
 private:
 
-	CNode(const CNode&);
+    CNode(const CNode&);
     void operator=(const CNode&);
 
-	// Network usage totals
-	 static CCriticalSection cs_totalBytesRecv;
-     static CCriticalSection cs_totalBytesSent;
-     static uint64_t nTotalBytesRecv;
-     static uint64_t nTotalBytesSent;
- 
+    // Network usage totals
+    static std::atomic<uint64_t> nTotalBytesRecv;
+    static std::atomic<uint64_t> nTotalBytesSent;
```

Comment:
I don't t hing  there should be a variable definition in header file. It will be duplicated when the file is included.